--- .profile	2020-09-13 15:47:52.000000000 +0200
+++ /home/jonathan/.profile	2020-09-15 18:25:39.470000000 +0200
@@ -41,3 +41,10 @@
 alias _!='sudo !!'
 alias ..='cd ..'
 alias ~='cd ~'
+
+export SSH_AUTH_SOCK=$HOME/.ssh/agent.sock
+ss -a | grep -q $SSH_AUTH_SOCK
+if [ $? -ne 0   ]; then
+    rm -f $SSH_AUTH_SOCK
+    ( setsid socat UNIX-LISTEN:$SSH_AUTH_SOCK,fork EXEC:"$HOME/c/npiperelay/npiperelay.exe -ei -s //./pipe/openssh-ssh-agent",nofork & ) >/dev/null 2>&1
+fi
